unit-test/addTask.test.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * @jest-environment jsdom
+ */
+import { addTask } from './addTask.js';
+import { dom } from './dom.js';
+import { storageData } from './storage.js';
+
+// jest.mock('../src/controllTools.js')
```

Comment:
- [ ] Please do not leave unused code as a comment in your codebase. this will make your code much noisy, It is always a good practice to keep your code clean from such comments.